### DISCUSSION TITLE
Browser based app?

### DISCUSSION THREAD

**Discussion by jolly47roger** [mapsforge/mapsforge#1587] [Q&A] [answered]
I have made a Mapsforge map of southern Britain using a variety of open sources of data. The map itself is about 1GB.

I want to offer a browser based app (like Leaflet or similar) that displays features up to zoom levels 18 or 19. Pre-rendering map tiles would produce the order of 100million tiles which is unmanageable.

I can't see a way to execute the tile server in JAVA within a webserver.

Or am I missing something obvious?

**Comment by devemux86** [comment, marked as answer]
@jolly47roger There some tile server implementations, check the [applications list](https://github.com/mapsforge/mapsforge/blob/master/docs/Mapsforge-Applications.md).
